src/components/Textarea.js

```diff
@@ -0,0 +1,38 @@
+import { h } from 'preact'
+import styled, { withTheme } from 'styled-components'
+import hoc from '../hoc'
+import { px } from '../utils'
+import { colors, radius } from '../theme'
+import { compose } from 'recompose'
+
+const Base = styled.textarea`
+  border-radius: ${px(radius)};
+  font-size: inherit;
+  border: 1px solid ${colors.gray.main};
+  padding: 8px 16px;
+  resize: vertical;
+  display: block;
+  width: 100%;
+
+  &:hover {
+    box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.1);
+  }
+  &::-webkit-input-placeholder {
```

Comment:
Styled components should handle browser prefixes, do we need this in this case?

Reply:
Yep I just checked and it's auto prefixed, I'll change these to `::placeholder` 👍